homeassistant/components/notify/kodi.py

```diff
@@ -4,76 +4,99 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/notify.kodi/
 """
+import asyncio
 import logging
+
+import aiohttp
 import voluptuous as vol
 
-from homeassistant.const import (ATTR_ICON, CONF_HOST, CONF_PORT,
-                                 CONF_USERNAME, CONF_PASSWORD)
-from homeassistant.components.notify import (ATTR_TITLE, ATTR_TITLE_DEFAULT,
-                                             ATTR_DATA, PLATFORM_SCHEMA,
-                                             BaseNotificationService)
+from homeassistant.const import (
+    ATTR_ICON, CONF_HOST, CONF_PORT, CONF_USERNAME, CONF_PASSWORD, CONF_SSL)
+from homeassistant.components.notify import (
+    ATTR_TITLE, ATTR_TITLE_DEFAULT, ATTR_DATA, PLATFORM_SCHEMA,
+    BaseNotificationService)
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
 import homeassistant.helpers.config_validation as cv
 
+REQUIREMENTS = ['jsonrpc-async==0.4']
+
 _LOGGER = logging.getLogger(__name__)
-REQUIREMENTS = ['jsonrpc-requests==0.3']
 
 DEFAULT_PORT = 8080
+DEFAULT_SSL = False
```

Comment:
I'd prefer not to, because then we're implying it's a proxy configuration, and there aren't any other proxy keys. Maybe we should get some more opinions? I'll go along with the consensus, but this would be a break from existing conventions.